bentoml/_internal/server/grpc/interceptors/__init__.py

```diff
@@ -68,47 +68,20 @@ async def intercept_service(
         handler_factory, next_handler = get_rpc_handler(handler)
         method_name = handler_call_details.method
 
-        if handler.response_streaming:
-
-            async def invoke_intercept_streaming(
-                request: Request, context: BentoServicerContext
-            ) -> t.AsyncGenerator[Response, None]:
-                coroutine_or_asyncgen = await self.intercept(
-                    next_handler, request, context, method_name
-                )
-
-                # Async server streaming handlers return async_generator, because they
-                # use the async def + yield syntax. However, this is NOT a coroutine
-                # and hence is not awaitable. This can be a problem if the interceptor
-                # ignores the individual streaming response items and simply returns the
-                # result of method(request, context). In that case the interceptor IS a
-                # coroutine, and hence should be awaited. In both cases, we need
-                # something we can iterate over so that THIS function is an
-                # async_generator like the actual RPC method.
-                if asyncio.iscoroutine(coroutine_or_asyncgen):
-                    asyncgen = await coroutine_or_asyncgen
-                    # If a handler is using the read/write API, it will return None.
-                    if not asyncgen:
-                        return
-                else:
-                    asyncgen = coroutine_or_asyncgen
-
-                async for r in asyncgen:
-                    yield r
-
-            return invoke_handler_factory(
-                invoke_intercept_streaming, handler_factory, handler
-            )
```

Comment:
The reason for removing this is supporting streaming RPC will bring more technical debt for this. We don't want to make streaming API for gRPC anyways so having to intercept streaming RPCs seems pointless.